doc/start/os-recommendations.rst

```diff
@@ -68,6 +72,8 @@ non-Linux systems but these are not supported by the core Ceph effort.
 +---------------+----------------+---------------+------------------+------------------+------------------+
 | Ubuntu 22.04  | A              |    A          |                  |                  |                  |
 +---------------+----------------+---------------+------------------+------------------+------------------+
+| MS Windows    | C (client only)|C (client only)| C (client only)  | C (client only)  |                  |
```

Comment:
"C" stands for "Ceph provides packages only. No tests have been done on these releases." which I interpret as something that is available on https://download.ceph.com, albeit untested.  We don't do this for Windows -- instead there is an MSI bundle that can be downloaded from https://cloudbase.it/ceph-for-windows/ and on top of that for the CephFS client Dokany needs to be obtained separately (i.e. it's not included there).  Should this get a new code/letter?